src/app/root-content/dashboard/widgets/widgets.component.scss

```diff
@@ -0,0 +1,77 @@
+:host {
+	display: block;
+	.ui.cards .ui.card.widget-container {
+		box-shadow: none;
+	}
+
+	.ui.fluid.card {
+		font-size: .95em;
+		box-shadow: none;
+	}
+
+	.ui.card.error {
+		/deep/ .ui.table, /deep/ .ui.header {
+			color: gray;
+		}
+		/deep/ .widget-text {
+			flex: auto;
+			color: gray;
+
+		}
+
+		/deep/ .widget-header {
+			background-color: #bcbec0;
+			padding: .5em .5em;
+		}
+	}
+
+	.ui.cards {
+		.ui.card {
+			font-size: .95em;
+			width: 245px;
+
+			/deep/ .widget-header {
+				background-color: orange;
+				padding: .5em .5em;
+			}
+
+			/deep/ .widget-button {
+				cursor: pointer;
+			}
+
+			/deep/ .widget-body {
+				display: flex;
+				flex-direction: row;
+			}
+
+			/deep/ .widget-icon {
+				height: 60px;
+			}
+
+			/deep/ .widget-text {
+				flex: auto;
+				color: black;
+				line-height: normal;
+				font-size: 50px;
+			}
+		}
+
+		.ui.card.error {
+			/deep/ .widget-text {
+				flex: auto;
+				color: gray;
+				line-height: normal;
+				font-size: 50px;
+			}
+			/deep/ .widget-header {
+				background-color: #bcbec0;
+				padding: .5em .5em;
+			}
+		}
+	}
+
+	/deep/ .ui.card.hidden {
+		display: none;
+	}
+
+}
```

Comment:
The reason behind having one SCSS file for all the widgets is that it is a safe way of ensuring that all the widgets of the same type have the same styles. Notice that the instance-widget.component has its own SCSS because it is a special type of widget that has its own design. However both document and triples widgets are the same type of widget that share a global SCSS, ideally if more widgets are added they will follow the same design and in the same way if a change in the design is made we could do it in that global SCSS, having a SCSS for each widget will mean repeating files and that if something changes in the style of the widgets, the change must be replicated manually for all the other widgets. @MiguelAraCo we were hoping for your input in this issue, let us know your thoughts